client/src/components/Dropdown/_Dropdown.scss

```diff
@@ -98,4 +99,26 @@
     bottom: 0px;
     margin-bottom: 4.2rem;
   }
-}
\ No newline at end of file
+
+  & .right-centered {
+    margin-left: 8.5rem;
+    top: -1.5rem;

```

Comment:
Can you change this `top` property so it's something dynamic so this is always vertically centered regardless of the number of dropdown items? I don't know how to do that without some research, but I think if you search Stack Overflow for "vertically center with absolute positioning" there's a way to do that.

Reply:
Had to dust off the CSS chops for this one for sure haha

Reply:
nice!